web/docusaurus.config.js

```diff
@@ -56,12 +56,16 @@ module.exports = {
       },
       items: [
         {
-          to: "docs/",
-          activeBasePath: "docs",
-          label: "Docs",
-          position: "left",
+          type: 'docsVersion',
+          position: 'left',
+          label: 'Docs',
           className: "navbar-item-docs navbar-item-outside",
```

Comment:
I replaced the old "Docs" link in Navbar with this new one, which is made by Docusaurus, and is smarter with regards to versions, it respects current version of docs. So it looks and the same but is better.